rules/manager_test.go

```diff
@@ -407,112 +407,118 @@ func TestForStateRestore(t *testing.T) {
 
 	// Prometheus goes down here. We create new rules and groups.
 	type testInput struct {
+		name            string
 		restoreDuration time.Duration
 		alerts          []*Alert
```

Comment:
[nit] Should we rename it to `expectedAlerts` for clarity?

Reply:
Done in https://github.com/prometheus/prometheus/pull/13980/commits/05ca082b079bb12dc7ad0f8c3940774df803401b